learnr_bank/learnr_set1.Rmd

```diff
@@ -0,0 +1,201 @@
+---
+title: "Question Bank Set 1"
+author: "Shemaiah Sotrah"
+date: "6/24/2020"
+output: learnr::tutorial
+runtime: shiny_prerendered
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(tidyverse)
+library(learnr)
+```
+
+# Resources for Shemaiah:
+
++ **Before you start**, go through this tutorial carefully: [learnr exercises](https://rstudio.github.io/learnr/exercises.html). 
+
++ For these questions, you do not need a setup chunk (but we will once we start using packages!)
++ Each question should have a corresponding `solution` chunk. IN that chunk, do not just type the answer - write out the code to get the answer (e.g. for q1 don't put `30`, put `5*6`)
++ Each question should have a corresponding hint. In this case, all hints are not actually code, but markdown text!! Therefore, you actually need to use HTML to write the hints instead of writing code chunks for hints (described in link above). 
+  + If a hint is ever code, use a chunk instead.
++ I've done set 1 question 1 below to show you exactly what I'm looking for.
+
+
+### Set 1: Using R as a calculator
+
+1. What is 5 times 6?
+  + **Hint:**  Use the asterisk `*` to multiply numbers
+  + Solution: 5*6
+  
+```{r calc_q1, exercise=TRUE}
+```
```

Comment:
At a minimum, I think we need a whole LINE inside this chunk. i.e, line 32 opens, line 33 is blank, and line 34 closes the chunk. This gets the solution to NOT appear, but it also leaves only a Hint button and no Solution button.

Reply:
AH!! Ok, "bad" news - it is NOT POSSIBLE to have both a hint and solution button - only 1! Since the hint is written after the solution, the hint overrides the solution button.

Reply:
Aww that's really too bad! What is your preference? Would you rather have a hint or a solution?